.github/workflows/deploy-site.yml

```
@@ -0,0 +1,44 @@
+name: Deploy site
+
+on: push
+
+jobs:
+  deploy-site:
+    name: Deploy site
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 7.9.5
```

`version` is no longer required as of `2.2.2`

https://github.com/pnpm/action-setup#version

```suggestion
      - uses: pnpm/action-setup@v2.2.2
```